Frends.AzureBlobStorage.ReadBlob/CHANGELOG.md

```diff
@@ -2,4 +2,8 @@
 
 ## [1.0.0] - 2022-04-08
 ### Added
-- Initial implementation
\ No newline at end of file
+- Initial implementation
+
+## [1.0.1] - 2022-04-21
+### Added
```

Comment:
This should say `Changed`, since you didn't add anything new, but modified existing behavior. In addition, the change should be above the last version, not below.